how-to/register-with-browser/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "openfin-workspace--register-with-browser",
+  "version": "6.0",
+  "description": "OpenFin Workspace -- Register with browser",
+  "main": "index.js",
+  "scripts": {
+    "dos": "desktop-owner-settings.bat && (npm run -s kill:fin || true) && (npm run -s kill:rvm || true)",
+    "build": "npm run build-server & npm run build-client",
+    "build-server": "tsc --project ./server",
+    "build-client": "webpack build --config ./client/webpack.config.js --mode=development",
+    "build-client-prod": "webpack build --config ./client/webpack.config.js --mode=production",
+    "start": "npm run server",
+    "client": "start fin://localhost:8080/manifest.fin.json",
+    "server": "node ./server/build/index.js",
+    "kill": "(npm run -s kill:fin || true) && (npm run -s kill:rvm || true)",
+    "kill:fin": "cmd.exe /c taskkill /F /IM OpenFin.exe /T",
+    "kill:rvm": "cmd.exe /c taskkill /F /IM OpenFinRVM.exe /T"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@openfin/workspace": "6.1.6",
+    "@openfin/workspace-platform": "6.1.6",
+    "uuid": "^8.3.2"
```

Comment:
We don't need to import this.

You can use: let uuid = window.crypto.randomUUID();

Reply:
removed uuid in favor of crypto, had to update to ts version 4.6.2